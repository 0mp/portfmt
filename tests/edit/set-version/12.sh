${PORTEDIT} set-version 0.61-335-gdf58b85 12.in | diff -L 12.expected -L 12.actual -u 12.expected -
