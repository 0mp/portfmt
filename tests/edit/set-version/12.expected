DISTVERSION=	0.61-335
DISTVERSIONSUFFIX=	-gdf58b85

include Makefile.configure
